sql/hypertable_index_triggers.sql

```diff
@@ -38,7 +38,6 @@ DECLARE
   hypertable_row _timescaledb_catalog.hypertable;
 BEGIN
     IF TG_OP = 'UPDATE' THEN
-        PERFORM _timescaledb_internal.on_trigger_error(TG_OP, TG_TABLE_SCHEMA, TG_TABLE_NAME);
```

Comment:
Do you mean checking that all the other columns are not changed ? I considered it but AFAIK it would mean a big if statement comparing individual columns. Also, as this is internal stuff I am not sure we need that level of security but I can add it if you feel strongly about it.